OWNERS_ALIASES

```diff
@@ -0,0 +1,35 @@
+aliases:
+    platform:
+    - zdtsw
+    - etirelli
+    - LaVLaS
+    - VaishnaviHire
+    - AjayJagan
+    - ykaliuta
+    - ajaypratap003
+    - mattmahoneyrh
+    - asanzgom
+    serving:
+    - VedantMahabaleshwarkar
+    - israel-hdez
+    - danielezonca
+    ide:
+    - harshad16
+    dashboard:
+    - andrewballantyne
+    - lucferbux
+    datasciencepipelines:
+    - HumairAK
+    modelregistry:
```

Comment:
@dhirajsb should we have at least one more owner?

Reply:
@dhirajsb Can you recommend anyone from model registry that we can add as reviewers?

Reply:
@tarilabs and @rareddy can be added as model-registry reviewers.